core/vpgl/vpgl_utm.h

```diff
@@ -56,7 +56,7 @@ class vpgl_utm
                double& x_out, double& y_out) const;
 
  private:
-   double a_{6378137}, b_{6356752.3142};
+   double a_{6378137}, b_{6356752.31424518};
```

Comment:
Did you mean to include this in this PR?  (Also, seems odd to increase precision of `b_` but not `a_`)

Reply:
My understanding is that we had some discrepancies between our calculations and other's and this precision increase was the fix to correct that. It could be put into a separate PR if necessary

Reply:
Ideally unrelated changes should go in separate PRs but this seems innocuous enough..